rllib/examples/env/pendulum_mass.py

```diff
@@ -1,25 +1,30 @@
 import numpy as np
 import gym
 from gym.envs.classic_control.pendulum import PendulumEnv
+from ray.rllib.env.meta_env import MetaEnv
+from ray.rllib.utils.annotations import override
 
 
-class PendulumMassEnv(PendulumEnv, gym.utils.EzPickle):
+class PendulumMassEnv(PendulumEnv, gym.utils.EzPickle, MetaEnv):
     """PendulumMassEnv varies the weight of the pendulum
 
     Tasks are defined to be weight uniformly sampled between [0.5,2]
     """
 
+    @override(MetaEnv)
     def sample_tasks(self, n_tasks):
         # Mass is a random float between 0.5 and 2
         return np.random.uniform(low=0.5, high=2.0, size=(n_tasks, ))
 
+    @override(MetaEnv)
     def set_task(self, task):
```

Comment:
Here and elsewhere, you can remove the pydoc since the overriden metaenv method is documented.